spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/mongo/MongoReactiveAutoConfigurationTests.java

```diff
@@ -106,6 +106,37 @@ public void optionsSslConfig() {
 				.isSameAs(this.context.getBean("myStreamFactoryFactory"));
 	}
 
+	@Test
+	public void createCustomize() {
```

Comment:
What does this test do? Perhaps you should check that the customizer has been invoked? The method name could be improved as well.